### DISCUSSION TITLE
Render animation frames to disk programmatically?

### DISCUSSION THREAD

**Discussion by ohlmic** [coldtype/coldtype#99] [Q&A] [answered]
Hi,

I'm glad I found this awesome library!

I've been trying to find a way to render the frames of the animation with code instead of pressing 'a' for example. 

I found in the docs that you also can type 'a' in the terminal (that doesn't work for me though for some reason), but I would like to use code in the Python source file instead.

Is this possible?

**Comment by stenson** [comment, marked as answer]
Thanks for the question — I'm not quite sure I understand, though. You can render all the frames of an animation by starting it on the command line with an `-a` flag (i.e. `coldtype myprogram.py -a`) and that should render all the frames of the animation without opening the viewer window. Is that what you have in mind?

This discussion might be a helpful reference as well, there was a lot of back and forth: https://github.com/goodhertz/coldtype/discussions/84

**Reply by ohlmic** [reply]
(REALLY late reply here)

Thanks! The #84 discussion was really helpful, I think the OP and I had a lot similar use cases in mind actually, so thanks for pointing me there!
